ci/docker_push.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 set -e
+set -x
 
 if [ "$TEST_TYPE" == "normal" ]
 then
   # this is needed to verify the example images
   docker build -f ci/Dockerfile-envoy-image -t lyft/envoy:latest .
+  make -C ci/build_alpine_container
```

Comment:
Roger 👍 I'll change this so that it builds the alpine image only in the want_push=true block.